include/fmt/chrono.h

```diff
@@ -878,7 +878,7 @@ template <typename T>
 struct has_member_data_tm_gmtoff<T, void_t<decltype(T::tm_gmtoff)>>
     : std::true_type {};
 
-#if defined(_WIN32)
+#if defined(_WIN32) && (!defined(WINAPI_FAMILY) || (WINAPI_FAMILY == WINAPI_FAMILY_DESKTOP_APP))
```

Comment:
Let's merge `tzset_once` into `format_utc_offset_impl` to avoid repeating the check and since it's only used there.

Reply:
@vitaut, after the proposed merge the ``_tzset`` function will be called multiple times. Once for each ``tm_writer`` template instantiation.
It won't throw an error. But there may be a slowdown if present many  ``tm_writer`` template instantiation with differents args.

Reply:
OK, let's keep this function but introduce a macro `FMT_USE_TZSET` instead to avoid repetition.